schema-guide.md

```diff
@@ -50,11 +50,15 @@ title: My Research Software
 version: 0.11.2
 ```
 
-### Transitive credit
+### Referencing other work
 
 When your software or data builds on what others have already done, it is good practice to add a `references` section to your
-`CITATION.cff` file. This way, whenever your work gets credited, a little bit of that goes into crediting the works that
-you built on.
+`CITATION.cff` file. This way, you give credit to those on whose work you build your own.
```

Comment:
yea I struggled with this one as well in the designs of the revamped cffinit website.

maybe something like
```suggestion
`CITATION.cff` file. This way, you give credit to other works on which your own work is built.
```

Reply:
Hm, you don't credit works though, but (ideally) people. Also I've realized that both examples suggest that your own work is *completely* built on other work, which seems to neglect own creativity in building a work. How about this:

```suggestion
`CITATION.cff` file. This way, you give credit to the authors of works that your own work builds on.
```

This way you credit the authors, and also "your own work builds on" suggests that you add a lego piece on top of an existing tower?

Reply:
works for me, I'll commit the suggestion